Prog/Tabulate householder income for projection.sas

```diff
@@ -0,0 +1,245 @@
+ Version:  SAS 9.2
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  Produce detailed popualtion by age group, race ethnicity and jurisciation from 2008-2017
+ ACS IPUMS data for the COGS region:
+ DC (11001)
+ Charles Couty(24017)
+ Frederick County(24021)
+ Montgomery County (24031)
+ Prince George's County(24033)
+ Arlington County (51013)
+ Fairfax County (51059)
+ Loudoun County (51107)
+ Prince William County (51153)
+ Alexandria City (51510)
+ Fairfax City (51600)
+ Falls Church City (51610)
+ Manassas City (51683)
+ Manassas Park City (51685)
+
+ Modifications:
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( RegHsg)
+%DCData_lib( Ipums)
+
+proc format;
+
+	value agegroupnew
+	.n= 'Not available'
+	1='under 25 years old'
+	2= '25-45 years old'
+	3= '45-65 years old'
+	4='65+ years old';
+  value race
+   .n = 'Not available'
+    1 = 'White'
+    2 = 'Black'
+    3 = "All Other ";
+  value hispan
+     .n = 'Not available'
+    0 = 'Not Hispanic'
+    1 = 'Hispanic';
+ value racenew
+   .n = 'Not available'
+    1 = 'White non-Hispanic'
+    2 = 'Black non-Hispanic'
+    3 = "Hispanic "
+	4 = "All other non-Hispanic ";
+  value agegroup
+     .n = 'Not available'
+    1= "0-5 years old"
+	2= "5-9 years old"
+	3= "10-14 years old "
+	4="15-19 years old"
+	5="20-24 years old "
+	6="25-29 years old"
+	7="30-34 years old"
+	8="35-39 years old"
+	9="40-44 years old"
+    10="45-49 years old"
+    11= "50-54 years old"
+    12= "55-59 years old"
+    13= "60-64 years old"
+    14= "65-69 years old"
+    15= "70-74 years old"
+    16= "75-79 years old" 
+    17 = "80-84 years old"
+    18= "85+ years old";
+
+	value Jurisdiction
+    1= "District of Columbia"
+	2= "Charles County"
+	3= "Frederick County "
+	4="Montgomery County"
+	5="Prince George's County"
+	6="Arlington County"
+	7="Fairfax, Fairfax City, and Falls Church"
+	8="Loudoun County"
+	9="Prince William, Manassas, and Manassas Park"
+    10="City of Alexandria";
+
+	value newinc
+	1= "0-32,600 Dollars"
+	2= "32,600-54,300 Dollars"
+	3="54,300-86,880 Dollars"
+	4="86,880-10,8600 Dollars"
+	5="108,600-130,320 Dollars"
+	6="130,320-217,200 Dollars"
+	7="More than 217,200 Dollars";
+
+run;
+
+%macro householdinfo(year);
+
+
+data Household_&year. (where=(upuma in ("1100101", "1100102", "1100103", "1100104", "1100105", "2401600", "2400301", "2400302","2401001", "2401002", "2401003", "2401004", "2401005", "2401006", "2401007", "2401101", "2401102", "2401103", "2401104", "2401105", "2401106", "2401107", "5101301", "5101302", "5159301", "5159302", "5159303", "5159304", "5159305", "5159306", "5159307", "5159308", "5159309", "5110701", "5110702" , "5110703", "5151244", "5151245", "5151246", "5151255")));
+set Ipums.Acs_&year._dc Ipums.Acs_&year._md Ipums.Acs_&year._va;
+
+  if upuma in ("1100101", "1100102", "1100103", "1100104", "1100105") then Jurisdiction =1;
```

Comment:
@maggiesu0725 - we are repeating these lists and some of the code in a lot of places. can you create macros to  keep all of the data prep code in one place so that you'd just run %create_race; in a data step instead of copying and pasting the code. I'm afraid at some point we'll want to change something and then have to do it in 30 programs. If you don't have time to do this let me know.